### PR TITLE
Fix errors in the copyright file

### PR DESCRIPTION
**Pull request by tibetiroka**
The high-dpi counterpart of [8953](https://github.com/endless-sky/endless-sky/pull/9853).

This one includes a lot more removals, so please double-check everything carefully.

### CODE REVIEW

**Comment by warp-core** [comment]
Can the changes from endless-sky/endless-sky#7881 being applied here be separated from the rest of the stuff here?
(I'll do it myself if you'd rather not go through everything again.)

**Comment by tibetiroka** [comment]
It could probably be done, but I don't really see why?

**Comment by warp-core** [comment]
I thought it might help with some conflicts with other things, but on further investigation, it might not be an issue.

**Comment by tibetiroka** [comment]
We'll see, then. If it becomes an issue, feel free to decouple the changes.

File: copyright [hunk 1]
```diff
@@ -545,6 +632,20 @@ Copyright: Becca Tommaso (tommasobecca03@gmail.com)
...
+Files:
+ images/outfit/*battery?hai*
+ images/outfit/anti-missile?hai*
+ images/outfit/cooling?ducts?hai*
+ images/outfit/dwarf?core?hai*
+ images/outfit/fission?hai*
+ images/outfit/fusion?hai*
+ images/outfit/heavy?anti-missile?hai*
+ images/ship/mfury*
+ images/ship/mquicksilver*
```
Comment (warp-core) [inline copyright:644]: Should be in a different place:

https://github.com/endless-sky/endless-sky/blob/941639dc6f7a2abdb945a7b45740e77f9dbc9c32/copyright#L1032-L1041

The corresponding high-DPI block is:

https://github.com/endless-sky/endless-sky-high-dpi/blob/bc3f6f51ae4f64cfcbcc65ab0a51ee3464cafd0e/copyright#L1068-L1075

**Pull request merged**